spacewalk/uyuni-setup-reportdb/bin/uyuni-setup-reportdb-user

```diff
@@ -197,6 +197,15 @@ else ### INTERACTIVE=0
 
 fi
 
+ADMIN_USERS=$(echo "SELECT usename FROM pg_catalog.pg_user WHERE usesuper='t' " | spacewalk-sql -i | tail -n +3 | head -n -2 | tr '[:upper:]' '[:lower:]')
```

Comment:
`spacewalk-sql -i` is using the "interactive" mode. This is not what you want.
I think you want `spacewalk-sql --select-mode`